avocado-setup.py

```diff
@@ -189,6 +189,29 @@ def get_avocado_bin():
         return avocado_binary
 
 
+def pip_install():
+    """
+    install package using pip
+    """
+    logger.info("install packages via pip interface")
+
+    if (sys.version_info >= (2, 0) and sys.version_info < (3, 0)):
+        pip_cmd = 'pip2'
+    else:
+        pip_cmd = 'pip3'
```

Comment:
can it be
`pip_cmd = sys.version_info[0]` ?

Reply:
are you referring that we can have like 
```if sys.version_info[0] =='2' :
    pip_cmd = 'pip2'
else:
   pip_cmd = 'pip3'```

Reply:
sorry, I meant,
`pip_cmd = "pip%s" % sys.version_info[0]`

Reply:
ah sure , thanks will adopt